File: Repo/Ecom.Core.Data/Models/Product.cs
using System;

namespace Ecom.Core.Data.Models
{
    public class Product
    {
        public Guid ProductId { get; set; }
        public string SkuCode { get; set; }
        public string Name { get; set; }
        public decimal Price { get; set; }
        public string ImageUrl { get; set; }
        public bool IsActive { get; set; }
        public string CreatedBy { get; set; }
        public string UpdatedBy { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime UpdatedOn { get; set; }
    }
}

File: Repo/Ecom.Core.ngApi/ClientApp/src/app/models/Product.ts
import { Guid } from "guid-typescript";

export interface Product {
    ProductId: string;
    SkuCode: string;
    Name: string;
    Price: number;
    ImageUrl: string;
    IsActive: boolean
    CreatedBy: string;
    UpdatedBy: string;
    CreatedOn: Date;
    UpdatedOn: Date;
}

export interface CartProduct extends Product{
    Quantity: number;
    TotalAmount:number
}
